src/tuner/nccl_ofi_regions.cpp

```diff
@@ -1167,6 +1177,74 @@ static ncclResult_t region_init_internal_p6(nccl_ofi_tuner_region_context_t *reg
 }
 
 
+static uint64_t calculateChunkSizeTreeLL128(uint64_t message_size, int nChannels, int nnodes)
```

Comment:
Same as for the static variables above, I think adding comments describing the steps of this chunk size calculation and where its coming from would help new-comers to this code get more context (e.g. is the `1 + log2(nnodes) + 0.1 * ppn` equation for `nStepsLL128` coming from NCCL's implementation of the tree algorithm? Why do we need to do the chunk size reduction? What is grain size vs chunk size?).